libs/pageserver_api/src/config.rs

```diff
@@ -678,14 +678,14 @@ pub mod tenant_conf_defaults {
     pub const DEFAULT_COMPACTION_TARGET_SIZE: u64 = 128 * 1024 * 1024;
 
     pub const DEFAULT_COMPACTION_PERIOD: &str = "20 s";
-    pub const DEFAULT_COMPACTION_THRESHOLD: usize = 10;
+    pub const DEFAULT_COMPACTION_THRESHOLD: usize = 8;
```

Comment:
Should we keep this at 10? A large threshold has two benefits: shorter and more compact L1 layers (less overlap), and longer time between L0 preemption.

Then again, I guess it helps reduce L0 read amp.

Reply:
The upper bound config can be adjusted in console for specific tenants if we have a backlog of L0 compaction.